README.md

```diff
@@ -11,316 +11,264 @@
+These includes checks for validating your model's performance (e.g. identify weak segments), the data's 
+distribution (e.g. detect drifts or leakages), data integrity (e.g. find conflicting labels) and more.
+These checks results can be run manually (e.g. during research) or trigerred automatically (e.g. during CI
+and production monitoring) and enable automatically making informed decisions regarding your model pipelines' 
+production-readiness, and behavior over time.
+--->
+
+## 🧩 Components
+
+Deepchecks includes:
+- **Deepchecks Testing**
+  ([docs](https://docs.deepchecks.com/stable/?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=components)): 
+  - Built-in Checks & Suites for Tabular, NLP & CV (open source). [Quickstart](#---------deepchecks-testing-quickstart------)
+- **CI & Testing Management**
+  ([docs](https://docs.deepchecks.com/stable/general/usage/ci_cd.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=components)):
+  - Collaborating over test results and efficient iterations until 
+  model is production-ready and can be deployed (open source & managed offering). [Quickstart](#---------deepchecks-ci--testing-management-quickstart------)
+- **Deepchecks Monitoring**
+  ([docs](https://docs.deepchecks.com/monitoring/stable/?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=components)): 
+  - Tracking your deployed models behavior when in production (open source & managed offering). [ Quickstart](#---------deepchecks-monitoring-quickstart------)
+
+This repo is our main repo as all components use the deepchecks checks in their core. See the [Getting Started](#--getting-started) section for more information about installation and quickstarts for each of the components.
+If you want to see deepchecks monitoring's code, you can check out the 
+files in the [deepchecks/monitoring](https://github.com/deepchecks/monitoring) repo.
+
+
+<details close>
+   <summary>
+      <h2>
+         ✅ Deepchecks Checks
+      </h2>
+   </summary>
+
+- Each check tests for a specific potential problem. Deepchecks has many preimplemented checks for
+  finding issues with the model's performance (e.g. identifying weak segments), 
+  data distribution (e.g. detect drifts or leakages) and data integrity (e.g. find conflicting labels).
+- Customizable: each check has many configurable parameters, and custom checks can easily be implemented.
+- Can be run manually (during research) or triggered automatically (in CI processes or production monitoring)
+- Check results can be consumed by: 
+   - [Saving to HTML](https://docs.deepchecks.com/stable/general/usage/export_save_results.html) or [viewing](https://docs.deepchecks.com/stable/general/usage/showing_results.html) in Jupyter - for visual output for human analysis
+   - [JSON output](https://docs.deepchecks.com/stable/general/usage/export_save_results.html) - for processing with code
```

Comment:
Programmatic output (result.value) is the best option if you're working with deepchecks from python.